data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -57,8 +57,13 @@ find "${ARTIFACTS}/rendered-assets" -name "*kubeconfig*" -print0 | xargs -0 rm
 find "${ARTIFACTS}/rendered-assets" -name "*.key" -print0 | xargs -0 rm
 find "${ARTIFACTS}/rendered-assets" -name ".kube" -print0 | xargs -0 rm -rf
 
-
 # Collect cluster data
+GATHER_KUBECONFIG="/opt/openshift/auth/kubeconfig"
+if [[ -f "/etc/kubernetes/bootstrap-secrets/kubeconfig" ]]; then
+    echo "Using /etc/kubernetes/bootstrap-secrets/kubeconfig as KUBECONFIG"
```

Comment:
nit: you can DRY this up slightly by shifting this `echo` down one line and using:

```sh
echo "Using ${GATHER_KUBECONFIG} as KUBECONFIG"
```